File: frontend/src/app/models/entity/Detallelote.ts
export class Detallelote {
    constructor(
        public row               ?: number,
        public sodeid            ?: number,
        public soliid            ?: number,
        public fechavto          ?: string,
        public lote              ?: string,
        public boddestino        ?: number,
        public bodorigen         ?: number,
        public cantidad          ?: number,
        public cantidaddev       ?: number,
        public codmei            ?: string,
        public descripcion       ?: string,
        public meinid            ?: number,
        public meintiporeg       ?: string,
        public tipobodorigen     ?: string,
        public tipoboddestino    ?: string,
        public glscombo          ?: string
    ) { }
}

File: backend/Modulos/models/detallelote.go
package models

// Detallelote is...
type Detallelote struct{
	Sodeid            int 		`json:"sodeid"`
	Soliid            int 		`json:"soliid"`
	Fechavto          string	`json:"fechavto"`
	Lote              string 	`json:"lote"`
	Boddestino        int 		`json:"boddestino"`
	Bodorigen         int 		`json:"bodorigen"`
	Cantidad          int 		`json:"cantidad"`
	Cantidaddev       int 		`json:"cantidaddev"`
	Codmei            string 	`json:"codmei"`
	Descripcion       string 	`json:"descripcion"`
	Meinid            int 		`json:"meinid"`
	Meintiporeg       string 	`json:"meintiporeg"`
	Tipobodorigen     string 	`json:"tipobodorigen"`
	Tipoboddestino    string 	`json:"tipoboddestino"`
	Glscombo          string 	`json:"glscombo"`
}